preview-src/comment.tsx

```diff
@@ -64,6 +65,7 @@ export function CommentView(comment: Props) {
 		onMouseLeave={() => setShowActionBar(false)}
 	>{ ((canEdit || canDelete) && showActionBar)
```

Comment:
Let's update this to just `showActionBar` so you can quote reply even if you don't have permissions to edit or delete the comment